src/yuzu/configuration/configure_input_player_widget.cpp

```diff
@@ -0,0 +1,1474 @@
+    p.setBrush(colors.button2);
+    DrawRoundRectangle(p, right_sideview_center + QPointF(0, 85), 20.2f, 40.2f, 3);
+
+    // left sideview slider body
+    p.setBrush(colors.slider);
+    DrawRoundRectangle(p, left_sideview_center + QPointF(0, -6), 26, 227, 3);
+    p.setBrush(colors.button2);
+    DrawRoundRectangle(p, left_sideview_center + QPointF(0, 85), 20.2f, 40.2f, 3);
+
+    // Right Slider decorations
+    p.setPen(colors.outline);
+    p.setBrush(colors.slider_arrow);
+    DrawArrow(p, right_sideview_center + QPoint(0, 73), Direction::Down, 2.1f);
+    DrawArrow(p, right_sideview_center + QPoint(0, 85), Direction::Down, 2.1f);
+    DrawArrow(p, right_sideview_center + QPoint(0, 97), Direction::Down, 2.1f);
+    DrawCircle(p, right_sideview_center + QPointF(0, 17), 3.8f);
+
+    // Left Slider decorations
+    DrawArrow(p, left_sideview_center + QPoint(0, 73), Direction::Down, 2.1f);
+    DrawArrow(p, left_sideview_center + QPoint(0, 85), Direction::Down, 2.1f);
+    DrawArrow(p, left_sideview_center + QPoint(0, 97), Direction::Down, 2.1f);
+    DrawCircle(p, left_sideview_center + QPointF(0, 17), 3.8f);
+
+    // Right SR and SL buttons
+    p.setPen(colors.outline);
+    p.setBrush(colors.slider_button);
+    DrawRoundRectangle(p, right_sideview_center + QPoint(0, 47), 10, 22, 3.6f);
+    DrawRoundRectangle(p, right_sideview_center + QPoint(0, -62), 10, 22, 3.6f);
+
+    // Left SR and SL buttons
+    DrawRoundRectangle(p, left_sideview_center + QPoint(0, 47), 10, 22, 3.6f);
+    DrawRoundRectangle(p, left_sideview_center + QPoint(0, -62), 10, 22, 3.6f);
+
+    // Right SR and SL text
+    SetTextFont(p, 5.5f);
+    p.setPen(colors.outline);
+    p.rotate(-90);
+    p.drawText(QPointF(-center.y() - 5, center.x() + 3) + QPointF(-47, 162.5f), tr("SL"));
+    p.drawText(QPointF(-center.y() - 5, center.x() + 3) + QPointF(62, 162.5f), tr("SR"));
+    p.rotate(90);
+
+    // Left SR and SL text
+    p.rotate(90);
+    p.drawText(QPointF(center.y() - 5, -center.x() + 3) + QPointF(47, 162.5f), tr("SR"));
+    p.drawText(QPointF(center.y() - 5, -center.x() + 3) + QPointF(-62, 162.5f), tr("SL"));
+    p.rotate(-90);
+
+    // LED indicators
+    const float led_size = 5.0f;
+    const QPointF left_led_position = left_sideview_center + QPointF(0, -33);
+    const QPointF right_led_position = right_sideview_center + QPointF(0, -33);
+    int led_count = 0;
+    for (const auto color : led_color) {
+        p.setBrush(color);
+        DrawRectangle(p, left_led_position + QPointF(0, 11 * led_count), led_size, led_size);
+        DrawRectangle(p, right_led_position + QPointF(0, 11 * led_count), led_size, led_size);
+        led_count++;
+    }
+}
+
+void PlayerControlPreview::DrawLeftBody(QPainter& p, const QPointF center) {
+    std::array<QPointF, 84> left_joycon;
+    std::array<QPointF, 66> qleft_joycon_sideview;
+    std::array<QPointF, 40> qleft_joycon_trigger;
+    std::array<QPointF, 47> qleft_joycon_slider;
+    const float size = 1.78f;
```

Comment:
```suggestion
    constexpr float size = 1.78f;
```

Ditto for the other three constants